fe/fe-core/src/main/java/org/apache/doris/load/StreamLoadRecordMgr.java

```diff
@@ -121,13 +128,14 @@ public void replayFetchStreamLoadRecord(FetchStreamLoadRecord fetchStreamLoadRec
         for (Backend backend : backends.values()) {
             if (beIdToLastStreamLoad.containsKey(backend.getId())) {
                 long lastStreamLoadTime = beIdToLastStreamLoad.get(backend.getId());
-                LOG.info("Replay stream load bdbje. backend: {}, last stream load version: {}", backend.getHost(), lastStreamLoadTime);
+                LOG.info("Replay stream load bdbje. backend: {}, last stream load time: {}", backend.getHost(), lastStreamLoadTime);
```

Comment:
what if disable stream load the value for lastStreamLoadTime will be?

Reply:
In fact, even if the collection process of be is disabled. 
The stream load record of fe will still obtain stream load information regularly. 
It’s just that every time you get it is empty.

In fact, FE can also add another variable to stop obtaining stream load record. 
But there are too many variables. 
To avoid using complexity. 
I did not add this switch to fe.